### ISSUE TITLE
Hangs while deleting episode

### ISSUE THREAD

**Issue by Enqueued** [jeff-hughes/shellcaster#49]
When using the 'x' keybind to remove an episode the tui hangs and is unable to register any keypress.

Attempting to remove the episode manually from the download directory does not update on the db side as opening the tui again shows that it is still downloaded.

**Comment by jeff-hughes** [comment]
Indeed, I'm able to reproduce that on my end as well. The tricky bit is that it seems to not hang immediately (the UI updates to show the episode is no longer downloaded). So I'll have to dig in a bit and see if I can find out what's causing the issue. Thanks for flagging this!

(And yes, it's expected behaviour that manually deleting the file does not update the app; the app does not watch the directory for changes.)
